content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -413,27 +413,39 @@ When you don't need to attach additional information to the relation, you can mo
 
 ## MongoDB
 
-In MongoDB, m-n-relations are represented by:
+In MongoDB, many-to-many (M:N) relations are represented by:
 
-- Relation fields on both sides, include a `@relation` attribute on _each side_
+- Relation fields on both sides, include a `@relation` attribute on _each side_, with mandatory `fields` and `references` arguments
 - A scalar list of referenced IDs on each side, including the `@db.ObjectId` attribute
 
-The following example demonstrate an m-n relation between posts and categories:
+The following example demonstrate an M:N relation between posts and categories:
 
 ```prisma
 model Post {
   id          String     @id @default(auto()) @map("_id") @db.ObjectId
   categoryIDs String[]   @db.ObjectId
-  categories  Category[] @relation(fields: [categoryIDs])
+  categories  Category[] @relation(fields: [b_ids], references: [id])
 }
 
 model Category {
   id      String   @id @default(auto()) @map("_id") @db.ObjectId
   postIDs String[] @db.ObjectId
-  posts   Post[]   @relation(fields: [postIDs])
+  posts   Post[]   @relation(fields: [a_ids], references: [id])
 }
 ```
 
+### Validation rules
+
+For MongoDB, Prisma validates M:N relations with the following rules:
+
+- Both sides of the relation must be arrays (in the data model the fields categories and posts).
+- The `@relation` attribute must define `fields` and `references` arguments
+- The fields argument must have only one scalar field defined, this scalar field must be of an array type.
+- The references argument must have only one scalar field defined, this scalar field must exist on the referenced model and must be of the same type as the scalar field in the fields argument, but singular (no array).
+- The scalar field where references points to must be an id.
+- No referential actions are allowed in this kind of relations (yet, support only after GA)
+- The implicit M:N relations as seen in SQL databases are not allowed with MongoDB.
```

Comment:
If we want to add more details, we could say implicit many-to-many

- have a hidden join table
- don't have use the `fields` argument

Reply:
Thanks @Tom... so is it this: 

The implicit M:N relations as typically seen in SQL databases are not allowed with MongoDB. However, you can still use an implicit M:N relation with MongoDB, by using a hidden join table and not implementing the `fields` argument.